homeassistant/components/sabnzbd/strings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Sabnzbd",
+        "description": "If you need help with the configuration have a look here: https://www.home-assistant.io/integrations/sabnzbd/",
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key%]",
+          "name": "[%key:common::config_flow::data::name%]",
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]",
+          "path": "[%key:common::config_flow::data::path%]",
+          "ssl": "[%key:common::config_flow::data::ssl%]"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_api_key": "[%key:common::config_flow::error::invalid_api_key%]"
+    },
+    "abort": {
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]"
```

Comment:
Why is only a single instance allowed? I also didn't notice you using this error.